### DISCUSSION TITLE
Vercel is not using my custom domain when creating dynamic routes

### DISCUSSION THREAD

**Discussion by bradvatne** [vercel/community#451] [Help] [answered]
I'm using Next-Auth to handle Oauth2 logins, and the [...nextauth].js file creates a URL that should be:
https://balidogfinder.com/api/auth/signin/google

But instead is 
https://balidogfinder-a40benqwr.vercel.app/api/auth/signin/google

Any advice on how to handle this would be much appreciated!

Project link is here: https://github.com/bradvatne/bali-dog-finder

EDIT: So I needed to add the NEXTAUTH_URL to the Vercel environment variables, works great now.

**Comment by ackmanx** [comment, marked as answer]
Thanks for the edit on the solution, had the same issue!
